### DISCUSSION TITLE
What about sharing an account?

### DISCUSSION THREAD

**Discussion by eldijomo** [CatimaLoyalty/Android#2195] [Q&A]
I'm new here, so hope I don't break rules,

So I just found this open software and It looks great. But Me and my wife share the same passes. We have used multiple digital pass holders and always could share our passes over the internet by logging in with the same account. This is super handy but now I found that I had to export a zip and send it over the internet to the other phone and import it there. Not the simpel solution.

Is there an option to share an account over the internet, cloud or maybe by connecting to the same NAS ?

**Comment by TheLastProject** [comment]
There are a few open issues how to possibly implement syncing between devices (for example #20 and #791) but sadly no code exists to support anything like this yet. It's definitely a place I would welcome improvement in.

You can use the built-in share feature to share one or more cards without images (either press the "share" icon when viewing a card, or long-press one or more cards in the main screen following by the "share" icon) over any type of text chat system, which is pretty fast (as sender: tap card -> share -> choose recipient; as receiver: tap link -> click save) but it is indeed not as good as having any type of automatic multi-device sync.

I'm still hoping one day someone shows up to help write code to support this, as my own TODO list is way too full to get to this any time soon sadly (and given I only use Catima on a single device and don't add new cards that often, I have little personal need for this feature).

**Reply by eldijomo** [reply]
Are exported zip files encrypted?

**Reply by TheLastProject** [reply]
If you password protect them they use AES encryption with the default zip4j AES key length (256):

https://github.com/CatimaLoyalty/Android/blob/7a0894e4d287016087dfdbf1bf58827762e29202/app/src/main/java/protect/card_locker/importexport/CatimaExporter.java#L93
https://github.com/srikanth-lingala/zip4j?tab=readme-ov-file#aes-encryption
